tests/PactNet.Native.Tests/Verifier/NativePactBrokerOptionsTests.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using PactNet.Native.Verifier;
+using Xunit;
+
+namespace PactNet.Native.Tests.Verifier
+{
+    public class NativePactBrokerOptionsTests
```

Comment:
I would add edge cases like empty space, null arguments etc as combability tests.

For instance

[Theory]
[InlineData(null)]
[InlineData("")]
[InlineData("   ")] 
public void BasicAuthentication_Fails_WhenCalled_WithInvalidArguments(string authArg)
{
   //I verify here an exception is thrown for instance.
}

Reply:
Yeah I had a look and the amount of places we'd have to add this boilerplate is pretty overkill to be honest. I've added all the guard checks in by adding a specific internal extension method for it so that's probably good enough to cover this.